### PR TITLE
$fakeUSD registration for testing decimals

### PR DESCRIPTION
**Pull request by zwilling**
# Pull Request Template

## Description

$"fakeUSD" registration for testing decimals

## Type of change

- [x] Metadata related change

## Checklist:

- [x] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

**Comment by zwilling** [comment]
Dear maintainers,
Would it be possible to merge this PR and https://github.com/cardano-foundation/cardano-token-registry/pull/3633 soon?
To my knowledge, this PR meets the requirements specified in the wiki and README, passed the checks and is only waiting for a review and merge.
Please tell me if anything else is wrong with it.

**Comment by Kartiiyer12** [comment]
Hi @zwilling The asset name "FADA" impersonates name of "ADA" and could not be merged. I am checking for FUSD as well.

**Review by Kartiiyer12 — APPROVED** [review]

**Pull request merged**
